nemo/collections/asr/data/audio_to_text_lhotse.py

```diff
@@ -43,18 +44,91 @@ def output_types(self) -> Optional[Dict[str, NeuralType]]:
             'sample_id': NeuralType(tuple('B'), LengthsType(), optional=True),
         }
 
-    def __init__(self, tokenizer):
+    def __init__(self, tokenizer, token_sequence_format: str = None):
```

Comment:
It'd be good to move the changes in this file into a new class e.g. `CanaryDataset`. We'd then have to connect it with the transformer code in `collections/asr/models`.